pyclient/tests/test_console.py

```diff
@@ -9,6 +10,7 @@
 
 class ConsoleTestCase(BaseTestCase):
 
+    @unittest.skip("GH ticket filed.")
```

Comment:
Please add a ticket number for any skipped cases.

Reply:
will do